src/site/content/en/blog/rakuten/index.md

```diff
@@ -0,0 +1,166 @@
+date: 2022-08-19
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/uB1PUfGOXOhrqUBW3tfO.jpg
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/OLBDJ1YEaVWGkR5lqNyR.jpg
+alt: A banner image with the Rakuten 24 logo and branded artwork.
+authors:
+  - linhduong
+  - shogokashiwase
+  - hayounglee
+  - ryunosukeakiba
+tags:
+  - blog
+  - case-study
+  - web-vitals
+---
+
+[Rakuten 24](https://24.rakuten.co.jp/) is an online store that collaborates with both major multinational and domestic consumer goods manufacturers to offer a great variety of daily necessities including healthcare, beverage, pet supplies, baby products, and more. This store is provided by Rakuten Group, Inc.—a global leader in internet services, and is among the top performers on their digital marketplace platform in Japan.
+
+Understanding the impact of web performance on user experience, the Rakuten 24 team has been continuously measuring, optimizing and monitoring Core Web Vitals and other metrics.
+
+As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.
+
+After analyzing homepage data, Rakuten 24 found that a good LCP score can lead to an increase of up to **61.13% in conversion rate**, **26.09% in revenue per visitor**, and **11.26% in average order value**. A good FID score can lead to an increase of up to **55.88% in conversion rate**.
+
+To further correlate Core Web Vitals and business metrics, Rakuten 24 also ran an A/B test focused on optimizing Core Web Vitals and related metrics, and saw an improvement of **53.37% in revenue per visitor**, **33.13% in conversion rate**, **15.20% in average order value**, **9.99% in average time spent**, and a **35.12% reduction in exit rate**.
+
+## Highlight the opportunity
+
+Though optimizing web performance is a smart investment to improve user experience and business growth, Rakuten 24 team understands how difficult it can be to convince stakeholders to adopt Core Web Vitals and focus on web performance. They believe that showing stakeholders exactly what kind of Return on Investment (ROI) performance optimization can bring is the best way to get them on board.
+
+Being a relatively new and independent service, Rakuten 24 used their advantage of flexibility to take up the challenge. They believe that the result of their case study would help them make more data-oriented decisions in the future, as well as help other developers measure the impact of their work and convince their stakeholders that improving performance is worth the investment.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ij6fET9W23vj1IOb3e6O.jpg", alt="Example screenshots of Rakuten 24's homepage, with mobile device frames around each screenshot.", width="800", height="537" %}
+  <figcaption>
+    Some example screenshots of Rakuten 24's homepage.
+  </figcaption>
+</figure>
+
+## The approach they used
```

Comment:
Can we do something about these stacked headings? They don't clearly intro the section when you read this in the preview. I actually think you could move everything up a level, lose "The approach they used".

Then make an h2 of "Web performance optimization techniques" or "Approach taken for web performance optimization", the follow that with a sentence explaining these were the techniques used. 

Same for the analysing section. THat would move everything up a level and prevent these confusing stacked headings.

Reply:
@rachelandrew, I pushed a change that flattens the structure of the document. Let me know if it works for you.

@duongkhlinh, we decided to reorganize some headings due to the feedback Rachel had above. Please advise on whether this will work for you.

Reply:
@malchata that looks better to me.

Reply:
@malchata Flattening the structure works for me. However, I have 2 suggestions:

**1. Can you consider adding a sentence to the end of the "Highlight the opportunity" section?**
Something like "Find out how they did it in this post" or "Find out the approach they used below".
**Reason:** It feels a bit sudden to jump straight from "Highlight the opportunity" to "Optimize JavaScript and resources". Adding the above sentence would make the flow more natural in my opinion.

**2. Can you consider changing the title of the analyzing section as below?**
- "Performance measuring, analyzing, and monitoring" → "Performance measuring"
- "Analyzing" → "Performance Analyzing"
- "Monitoring" → "Performance Monitoring"
 
**Reason:** "Performance measuring, analyzing, and monitoring" section now only talks about measuring so changing it to "Performance measuring" seems easier to understand. Adding "performance" to "analyzing" and "monitoring" makes it clearer too I think.